test/e2e/fixtures.go

```diff
@@ -177,29 +177,33 @@ func exportLogs(tb testing.TB, data *TestData) {
 	}
 
 	// dump the logs for Antrea Pods to disk.
-	data.forAllAntreaPods(func(nodeName, podName string) error {
+	writePodLogs := func(nodeName, podName, nsName string) error {
 		w := getPodWriter(nodeName, podName, "logs")
 		if w == nil {
 			return nil
 		}
 		defer w.Close()
-		cmd := fmt.Sprintf("kubectl -n %s logs --all-containers %s", antreaNamespace, podName)
+		cmd := fmt.Sprintf("kubectl -n %s logs --all-containers %s", nsName, podName)
 		stdout := runKubectl(cmd)
 		if stdout == "" {
 			return nil
 		}
 		w.WriteString(stdout)
 		return nil
-	})
+	}
+	data.forAllNsPods("app=antrea", antreaNamespace, writePodLogs)
+
+	// dump the logs for monitoring Pods to disk.
+	data.forAllNsPods("", monitoringNamespace, writePodLogs)
 
 	// dump the output of "kubectl describe" for Antrea pods to disk.
-	data.forAllAntreaPods(func(nodeName, podName string) error {
+	data.forAllNsPods("app=antrea", antreaNamespace, func(nodeName, podName, nsName string) error {
```

Comment:
nit: I would suggest the name `forAllMatchingPodsInNamespace` instead for readability since: 1) one of the parameters is a label selector and 2) we don't usually abbreviate `namespace` to `ns` in these tests

Reply:
Done